sbin/build.sh

```diff
@@ -920,6 +920,62 @@ checkingToolSummary() {
    sed -n '/^Tools summary:$/,$p' "${inputConfigFile}" > "${outputConfigFile}"
 }
 
+# Determine FreeType version being used in the build from either the system or bundled freetype.h definition
+addFreeTypeVersionInfo() {
+   # Default to "system" the jdk8 only value
+   local FREETYPE_TO_USE="system"
+   if [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" != "${JDK8_CORE_VERSION}" ]; then
```

Comment:
should that be `==`? Seems different to the comment above

Reply:
!= is correct, the comment is missleading, i've changed it
jdk8 can only be system
jdk-11+ supports bundled or system